pkg/config/validation/validation.go

```diff
@@ -25,7 +25,6 @@ import (
 	"strings"
 	"time"
 
-	xdsUtil "github.com/envoyproxy/go-control-plane/pkg/wellknown"
```

Comment:
did these names change back?

Reply:
Yes, all those names changed. I made all of them literal at the previous SHA of go-control-plane. There is no need to import a changing package just to save on typing a constant.

Reply:
What happens on the next upgrade?, I suppose as long as don't use the wellknown package we are ok.

Reply:
We can do whatever want. At some point, istio needs to stop using the deprecated names and APIs.